File: Backend/MongoDBTestProject/Model/FuelQueueRemoveRequest.cs
namespace MongoDBTestProject.Model
{
    /** 
     * fuel queue remove request class for geth the request
     * **/
    public class FuelQueueRemoveRequest
    {
        public String EndDateTime { get; set; } = String.Empty;
        public String FuelAmount { get; set; } = String.Empty;
    }
}

File: Frontend/app/src/main/java/com/example/fuelqueueapplication/api/request/FuelQueueRemoveRequest.java
package com.example.fuelqueueapplication.api.request;

/**
 * fuel queue remove request class
 * **/
public class FuelQueueRemoveRequest {
    private String FuelAmount;
    private String EndDateTime;


    public FuelQueueRemoveRequest(String endDateTime, String fuelAmount) {
        EndDateTime = endDateTime;
        FuelAmount = fuelAmount;
    }
}
